ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -1101,7 +1099,7 @@ tests:
       SINGLE_NODE_AZURE_INSTANCE_TYPE: Standard_D32ps_v5
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-sno
+    workflow: cucushift-installer-rehearse-azure-ipi-sno-etcd-encryption
 - as: azure-ipi-sno-amd-f28-destructive
```

Comment:
Similar to https://github.com/openshift/release/pull/49845/files#r1524364786, once the `azure-ipi-sno-f28` get some change, accordingly `azure-ipi-sno-amd-f28-destructive` also need the same change.